ptx/S_12_5_Div.ptx

```diff
@@ -23,14 +23,9 @@
     	<caption> A vector field with changing strength </caption>
     	<image width="55%" source="images/fig_12_5_vf1"/>
     </figure>
-	
-
 	</introduction>
-
 	<exploration xml:id="Divergence_PA">
-		<introduction>
 			<p> In this preview activity, we will look at a few examples of vector fields and try to assess when the vector field has increased or decreased in strength on the interior of some region. </p>
-		</introduction>
 			<task xml:id="Div_PA_task1">
 				For <xref ref="Div_PA_task1">Question </xref>, use the following vector field plot to answer each question.
 				<figure xml:id="Div_PA_vf1">
```

Comment:
A `task` needs to have its content at least inside `p`. Since we ultimately want `answer` and `solution` and in some cases `hint` on these, you really need `<task><statement><p>`stuff`</p></statement></task>`. The lack of `p` here is what's causing some of the figure captions to show up on a different line than the figure number.

Reply:
I also apparently can't add a comment to line 35. PTX provides the `@label` attribute on `ol`. I think you want `<ol label="a">` for what you seem to be suggesting.